tests/components/roborock/conftest.py

```diff
@@ -43,6 +58,8 @@ def bypass_api_fixture() -> None:
         "roborock.api.AttributeCache.async_value"
     ), patch(
         "roborock.api.AttributeCache.value"
+    ), patch(
+        "homeassistant.components.roborock.image.asyncio.sleep"
```

Comment:
It's safer to patch the time to sleep. Define a constant with that time to make it easy to patch.